corehq/apps/userreports/pillow.py

```diff
@@ -273,13 +370,14 @@ class ConfigurableReportKafkaPillow(ConstructedPillow):
     # we could easily remove the class and push all the stuff in __init__ to
     # get_kafka_ucr_pillow below if we wanted.
 
-    def __init__(self, processor, pillow_name, topics, num_processes, process_num, retry_errors=False):
+    def __init__(self, processor, pillow_name, topics, num_processes, process_num, retry_errors=False,
+                checkpoint_frequency=1000):
```

Comment:
looks like `checkpoint_frequency` is always passed in and has a value of 100, which is confusing when default arg is different from the most common passed value